python/cudf_polars/cudf_polars/experimental/io.py

```diff
@@ -354,70 +358,283 @@ def _(
     return ir, {ir: PartitionInfo(count=1)}  # pragma: no cover
 
 
+class StreamingSink(IR):
```

Comment:
For consistency, should we rename `SplitScan` to `StreamingScan`?

Reply:
The "Split" in `SplitScan` refers to the fact that we are splitting large-file reads into multiple IR nodes - The streaming executor does **not** use any `SplitScan` nodes when the file(s) are small enough. The `StreamingSink` class serves a slightly different purpose, because it is **always** used by the streaming executor.